packages/bundle-source/test/no-transforms.test.js

```diff
@@ -0,0 +1,36 @@
+// @ts-check
+import test from '@endo/ses-ava/prepare-endo.js';
+
+import fs from 'fs';
+import url from 'url';
+import { decodeBase64 } from '@endo/base64';
+import { ZipReader } from '@endo/zip';
+import bundleSource from '../src/index.js';
+
+test('no-transforms applies no transforms', async t => {
+  const entryPath = url.fileURLToPath(
+    new URL(`../demo/circular/a.js`, import.meta.url),
+  );
+  const { endoZipBase64 } = await bundleSource(entryPath, {
+    moduleFormat: 'endoZipBase64',
+    noTransforms: true,
+  });
+  const endoZipBytes = decodeBase64(endoZipBase64);
+  const zipReader = new ZipReader(endoZipBytes);
+  const compartmentMapBytes = zipReader.read('compartment-map.json');
+  const compartmentMapText = new TextDecoder().decode(compartmentMapBytes);
+  const compartmentMap = JSON.parse(compartmentMapText);
+  const { entry, compartments } = compartmentMap;
+  const compartment = compartments[entry.compartment];
+  const module = compartment.modules[entry.module];
+  // Alleged module type is not precompiled (pre-mjs-json)
+  t.is(module.parser, 'mjs');
+
+  const moduleBytes = zipReader.read(
+    `${compartment.location}/${module.location}`,
```

Comment:
...I didn't see Windows in the build matrix, so not gonna complain about all this stuff that should probably be `path.join`'d or `path.normalize`'d

Reply:
We recently removed Windows from the build matrix for reasons, but `@endo/zip` only deals in Unixy paths, regardless of the host platform, so we’re good here.